pkg/operator/k8sutil/pod.go

```diff
@@ -264,7 +264,10 @@ func GetPodLog(clientset kubernetes.Interface, namespace string, labelSelector s
 		builder := &strings.Builder{}
 		defer readCloser.Close()
 		_, err = io.Copy(builder, readCloser)
-		return builder.String(), err
+		if err != nil {
+			return "", errors.Wrapf(err, "error copying file from %s to %s", builder, readCloser)
+		}
+		return builder.String(), err //nolint
```

Comment:
```suggestion
		return builder.String(), err //nolint: no else statement needed
```

Reply:
changing to `//nolint, no need for else statement`, using `:` will give an error because it is used to specify specific linter.